doc/tutorials/quickstart_in_rviz/launch/demo.launch.py

```diff
@@ -20,6 +20,13 @@ def generate_launch_description():
             description="RViz configuration file",
         )
     )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "rviz_tutorial",
+            default_value="False",
+            description="Tutorial flag for empty Rviz config",
```

Comment:
Can you include in the description whether true or false means empty vs. filled out config?

Reply:
Removed this variable as per @MikeWrock's suggestion.